packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -50,7 +50,7 @@ export class ActivityItem extends React.Component<IActivityItemProps, {}> {
 export type Alignment = 'start' | 'end' | 'center' | 'space-between' | 'space-around' | 'space-evenly' | 'baseline' | 'stretch';
 
 // Warning: (ae-forgotten-export) The symbol "React" needs to be exported by the entry point index.d.ts
-//
+// 

```

Comment:
I'm thinking that these trailing whitespaces might be an api-extractor-specific issue that I'll try to look into soon. In the meantime, if it's annoying, you can go into User Settings in VSCode, add the line

```json
"files.trimTrailingWhitespace": true
```

then open and save the api.md file and it should get rid of these!

Reply:
I'm not too bothered, should I do it for consistency?

Reply:
It's okay, I don't think it's super necessary! I'm hoping to upgrade office-ui-fabric-react's api-extractor version soon that should get rid of the issue soon.